src/service_manager.rs

```diff
@@ -21,6 +21,8 @@ bitflags::bitflags! {
 
         /// Can enumerate services or receive notifications.
         const ENUMERATE_SERVICE = Services::SC_MANAGER_ENUMERATE_SERVICE;
+
+        const ALL_ACCESS = Services::SC_MANAGER_ALL_ACCESS;
```

Comment:
Is there any suitable documentation you could add here? The more descriptive the better :)

Reply:
Updated, though I admit it's still a little vague. From MSDN:

> SC_MANAGER_ALL_ACCESS (0xF003F) | Includes STANDARD_RIGHTS_REQUIRED, in addition to all access rights in this table.

Reply:
Typically combining all of the flags in `ServiceManagerAccess` gives you (perhaps) full access to the SC panel and with `bitflags` that we use it's easy, just call `::all()` on the struct. I could be wrong but that worked well for most manipulations that we've done in the past.

Reply:
`::all()` currently doesn't do the job, but perhaps if we added the missing flags. It looks like the mask set in ALL_ACCESS has some flags that don't specifically fit within the current sets though.

Reply:
Thanks! This is good.